apps/financial-aid/web-veita/graphql/authLink.ts

```diff
@@ -1,18 +1,15 @@
 import { setContext } from '@apollo/client/link/context'
-import { CSRF_COOKIE_NAME } from '@island.is/financial-aid/shared/lib'
-import Cookie from 'js-cookie'
+import { getSession } from 'next-auth/client'
 
-export default setContext((_, { headers }) => {
-  const token = Cookie.get(CSRF_COOKIE_NAME)
+export default setContext(async (_, { headers }) => {
+  const session = await getSession()
 
-  if (token) {
-    return {
-      headers: {
-        ...headers,
-        authorization: `Bearer ${token}`,
-      },
-    }
+  return {
+    headers: {
+      ...headers,
+      authorization: session?.accessToken
+        ? `Bearer ${session.accessToken}`
+        : '',
+    },
   }
-
-  return headers
 })
```

Comment:
This is the same file as in web-osk can we maybe move this to libs? 🧎

Reply:
This is a good point, we could take most out of this folder and reuse it, but let's leave it for a different PR, if we decide to do it.